skypy/power_spectrum/_camb.py

```diff
@@ -7,7 +7,8 @@
 ]
 
```

Comment:
Let me suggest `from astropy import units`

Reply:
I disagree: `from astropy import units as u` is a prevalent idiom that is used everywhere in astropy, because `units.eV` is harder to parse than `u.eV`.

Reply:
I agree. However, I thought we agreed on avoiding this type of abbreviation?

Reply:
I wasn’t part of any discussion on this so I don’t know. I agree to not coming up with unconventional ones like `import subprocess as sbp` but I think the default ones like `np` and `u` are helpful.

Reply:
I have given reviews before suggesting not to use `import a as b` except for `import numpy as np` as I think that is the only one that can reasonably be considered a "convention" in the community. Astropy may use `import astropy.units as u` internally but that doesn't mean your average astronomer reading python code would know what `u` is without checking the imports. And I disagree I think `units.eV` is ***much*** clearer than `u.eV`. In pretty much all cases I feel it's not worth the loss of clarity to abbreviate names especially now that we allow more than 80 characters per line.